readthedocs/core/views.py

```diff
@@ -235,21 +237,30 @@ def github_build(request):
         except:
             # Generic post-commit hook
             obj = json.loads(request.body)
-        url = obj['repository']['url']
-        ghetto_url = url.replace('http://', '').replace('https://', '')
+        repo_url = obj['repository']['url']
+        hacked_repo_url = repo_url.replace('http://', '').replace('https://', '')
+        ssh_url = obj['repository']['ssh_url']
+        hacked_ssh_url = ssh_url.replace('git@', '').replace('.git', '')
         try:
             branch = obj['ref'].replace('refs/heads/', '')
         except KeyError:
             response = HttpResponse('ref argument required to build branches.')
             response.status_code = 400
             return response
-        pc_log.info("(Incoming GitHub Build) %s [%s]" % (ghetto_url, branch))
+
         try:
-            return _build_url(ghetto_url, [branch])
+            repo_projects = get_project_from_url(hacked_repo_url)
+            if repo_projects:
+                pc_log.info("(Incoming GitHub Build) %s [%s]" % (hacked_repo_url, branch))
+            ssh_projects = get_project_from_url(hacked_ssh_url)
+            if ssh_projects:
+                pc_log.info("(Incoming GitHub Build) %s [%s]" % (hacked_ssh_url, branch))
```

Comment:
Is there any reason it needs to? We give precedence below to `repo_projects`, seems we should move the logic into `if/elif` and log/build once. That is:

```python
if repo_projects:
    repo_url = hacked_repo_url
    projects = repo_projects
elif ssh_projects:
    repo_url = hacked_ssh_url
    projects = ssh_projects
pc_log.info("(Incoming GitHub Build) %s [%s]", repo_url, branch)
_build_url(repo_url, projects, [branch])
```